### PR TITLE
data: kumarankit999

### PR DESCRIPTION
**Pull request by kumarankit999**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6188"></a>

### CODE REVIEW

File: kumarankit999.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "Ankit Kumar",
+  "displayStatsPublic": true,
```
Comment (Ananya2001-an) [inline kumarankit999.json:3]: Please remove this. 
```suggestion
```

**Review by Ananya2001-an — CHANGES_REQUESTED** [review]
Please make the requested changes. Also move this file into the `data` folder.

**Review by harshsingh32 — APPROVED** [review]

**Review by harshsingh32 — COMMENTED** [review]
looks fine

**Comment by amandamartin-dev** [comment]
> looks fine

Hello! Please be sure to fully read all review comments before adding yours in the future.  This profile requires changes as noted by a previous reviewer so saying that it looks fine can lead to confusion for this contributor and the maintainers reviewing what is ready for merging.

**Comment by SaraJaoude** [comment]
Due to the length of time this PR has been open, I will merge and fix in a temporary branch.

**Comment by SaraJaoude** [comment]
Correction:

I merged this PR to a temporary branch as it needed to be fixed as it was not in the data folder. 

However I noticed that you already have a profile: https://github.com/EddieHubCommunity/LinkFree/blob/main/data/kumarankit999.json

Therefore I reverted the merge. 

You can only create one profile in the data directory.  Here are the docs to update your existing profile: https://linkfree.io/docs/how-to-guides/editing

**Pull request merged**